functions/venue.js

```diff
@@ -206,6 +206,8 @@ const createVenueData = (data, context) => {
   if (data.template === VenueTemplate.audience) {
     venueData.showReactions = data.showReactions;
 
+    venueData.showShoutouts = data.showShoutouts;
```

Comment:
I believe shout-outs are intended to be part of more templates than just the audience? At least they used to be (even though I believe they were hackily commented out from a bunch of them) So we may just want to do this in all cases rather than for this specific venue type?

Reply:
I think there is some variable in settings or something that has all the venues that use shoutouts. However, the showReactions is only used in the auditorium, so I only implemented showShoutouts for the auditorium as well. Should I add the showShoutouts and showReactions to the other venues that use reactions too?

Reply:
While more templates use shoutouts, the `HAS_REACTIONS_TEMPLATES` variable in `settings` only has "audience", so I'm only going to implement showShoutouts for the audience in this PR. (Also this is urgently needed for github so it's quicker if we just implement one thing)

<img width="517" alt="Screen Shot 2021-06-29 at 7 08 39 PM" src="https://user-images.githubusercontent.com/51083763/123846558-6ce9dc80-d90d-11eb-8ec6-9118c7ce1d56.png">

Reply:
> Should I add the showShoutouts and showReactions to the other venues that use reactions too?

Please! Definitely should be in all places.

Reply:
> (Also this is urgently needed for github so it's quicker if we just do this)

Creating extra work for ourselves to cleanup weirdly broken/deficient features in future, when it would take very little extra effort to fix it all properly here, doesn't really make sense tbh.